libs/remote_storage/src/azure_blob.rs

```diff
@@ -341,7 +341,7 @@ impl RemoteStorage for AzureBlobStorage {
             // to still be representable by signed 64 bit integers).
             // TODO remove workaround once the SDK adds open range support
             // https://github.com/Azure/azure-sdk-for-rust/issues/1438
-            let end_exclusive = u64::MAX << 4;
```

Comment:
`>> 4` it's not the same as `/ 4`. It equals to `/ 16`

Reply:
you are right but it doesn't really matter: my goal was to just have something sufficiently far away from i64::MAX that there is not going to be any bugs because of us being close to that number. Not saying that there *are* such bugs, just that they might exist and we just rather avoid them.